src/reboost/build_hit.py

```diff
@@ -205,41 +209,55 @@ def build_hit(
             buffer size for use in the `LH5Iterator`.
 
     """
-
     # extract the config file
     if isinstance(config, str):
-        config = io.load_dict(config)
+        config = utils.load_dict(config)
 
     # get the arguments
     if not isinstance(args, AttrsDict):
         args = AttrsDict(args)
+    time_dict = {
```

Comment:
I would define this data structure in a separate module and store there all the functions to manipulate it (e.g. printing to the console). Maybe it can be a small class that inherits from AttrsDict or just a named tuple. You can then override the `__repr__()` and/or `__str__()` methods to make them print a nice profiling summary